extensions/cornerstone/src/commandsModule.ts

```diff
@@ -1216,6 +1217,17 @@ function commandsModule({
         viewportGridService.getActiveViewportId()
       );
     },
+    deleteActiveAnnotation: () => {
+      const activeAnnotationsUID = cornerstoneTools.annotation.selection.getAnnotationsSelected();
+      const hasMoreThanOneAnnotationSelected = activeAnnotationsUID.length > 1;
+
+      if (hasMoreThanOneAnnotationSelected) {
+        return;
+      }
+
+      const activeAnnotationUID = activeAnnotationsUID[0];
+      cornerstoneTools.annotation.state.removeAnnotation(activeAnnotationUID);
+    },
```

Comment:
here grab the selected UIDs and loop over the other command that delete based on UID